doc/src/index.md

```diff
@@ -1,4 +1,8 @@
-# Julia Documentation
+# Julia 0.7 Documentation
+
+Welcome to the documentation for Julia 0.7.
+
+Please read the [release notes](NEWS.md) to see what has changed since Julia 0.6.
```

Comment:
`... what has changed since Julia 0.6.` -> `... what has changed since the last release.` ?